pkg/providers/imagefamily/azlinux.go

```diff
@@ -31,9 +31,11 @@ import (
 )
 
 const (
-	AzureLinuxGen2ImageDefinition    = "V2gen2"
-	AzureLinuxGen1ImageDefinition    = "V2"
-	AzureLinuxGen2ArmImageDefinition = "V2gen2arm64"
+	AzureLinuxGen2ImageDefinition      = "V2gen2"
+	AzureLinuxGen1ImageDefinition      = "V2"
+	AzureLinuxGen2ArmImageDefinition   = "V2gen2arm64"
+	AzureLinux2Gen2FIPSImageDefinition = "V2gen2fips"
+	AzureLinux2Gen1FIPSImageDefinition = "V2fips"
```

Comment:
Checked all the images vs the ListNodeImageVersions API results, and they look good:
https://github.com/Azure/karpenter-provider-azure/blob/deb71b4a03885a5294912a39fbcac0e9956150d4/pkg/fake/nodeimageversionsapi.go#L35